### ISSUE TITLE
Way to get log axes with datashader dsshow and matplotlib

### ISSUE THREAD

**Issue by rotheconrad** [holoviz/datashader#1087]
#### Is your feature request related to a problem? Please describe.
I cannot find a way to get dsshow to return the points with a log axis or convert them to a log axis with matplotlib.

#### Describe the solution you'd like
Better documentation with instructions or feature addition if it is not currently doable.

#### Describe alternatives you've considered
dsshow doesn't appear to be fully linked to other datashader functionality. Matplotlib controls do not work to manipulate the object returned by dsshow.

#### Additional context
I believe this doesn't depend on versions or any other context.

**Comment by jbednar** [comment]
Good point. Setting it up on the Datashader side should just require passing `x_axis_type='log'` and/or `y_axis_type='log'`when creating the Canvas in `datashader.mpl_ext.DSArtist.aggregate`. I don't know what's required on the Matplotlib side, though.  @nvictus or @tacaswell ?

> Matplotlib controls do not work to manipulate the object returned by dsshow.

Can you be more specific and provide examples? I'm not sure what's missing here.
